pom.xml

```diff
@@ -59,6 +52,10 @@
           <id>ome.maven</id>
           <url>http://artifacts.openmicroscopy.org/artifactory/maven/</url>
         </repository>
+        <repository>
```

Comment:
Shouldn't this be a temporary block in a separate PR which can be used to push the pom from staging to release?